collect.py

```diff
@@ -203,6 +203,18 @@ def screenshot(driver, file_path):
     image.save(file_path)
 
 
+def get_code(driver, file_path):
+    wait_loaded(driver)
+
+    with open(file_path, 'w', encoding='utf-8') as f:
+        f.write(driver.execute_script('return document.documentElement.outerHTML'))
+
+
+def get_screenshot_and_domtree(driver, file_name):
+    screenshot(driver, 'data/' + file_name + '.png')
+    get_code(driver, 'data/' + 'dom_LOCATION_' + file_name + '.txt')
```

Comment:
Don't actually put LOCATION here, it was just a placeholder to show you what the name should be 😄